UnitySDK/Assets/TiltBrush/Scripts/Editor/ModelImportSettings.cs

```diff
@@ -245,14 +250,16 @@ public class ModelImportSettings : AssetPostprocessor {
       LogWarningWithContext("Not hypercolor?");
     }
 
-    var data = new List<Vector3>();
+    var data = new List<Vector3>(mesh.vertexCount);
     mesh.GetUVs(uvSet, data);
-    for (int i = 0; i < data.Count; ++i) {
+    int dataCount = data.Count;
+    for (int i = 0; i < dataCount; ++i) {
       Vector3 tmp = data[i];
       tmp.z *= .1f;
       data[i] = tmp;
     }
     mesh.SetUVs(uvSet, data);
+    data.Clear();
```

Comment:
Does this really make a difference? the backing array will become garbage on line 263 anyway

Reply:
It is true that there's a bunch of speculation on my part and it may be primarily the inlining solving my problem, but some of the strange hangs we were seeing I assumed were more memory based as happen on longer imports somewhat erratically so I went hunting for places where memory could be an issue, so the addition of the .Clear() calls may be useless? I'm not totally sure how to profile import problems like that, and my bias is to distrust Lists and unity

I don't mind at all if you discard the lot, and put in whatever changes you want more cleanly yourself either. Not too sure how to resolve requests for changes as this is the first time trying to submit any code anywhere, so I'll see what submitting a pull request on top of this one does with changes.

Reply:
This is my first github code review as well, so I'm muddling through as well I can. I'll ping this again when my changes land.